sqlglot/transforms.py

```diff
@@ -308,11 +312,7 @@ def _unnest_zip_exprs(
     ) -> t.List[exp.Expression]:
         if has_multi_expr:
             if not unnest_using_arrays_zip:
-                if generator:
-                    generator.unsupported(
-                        f"Multiple expressions in UNNEST are not supported in "
-                        f"{generator.dialect.__module__.split('.')[-1].upper()}"
-                    )
+                raise UnsupportedError("Cannot transpile UNNEST with multiple arrays to Hive")
```

Comment:
This is dialect-agnostic:
```suggestion
                raise UnsupportedError("Cannot transpile UNNEST with multiple arrays")
```